frontend/src/features/properties/filter/PropertyFilter.tsx

```diff
@@ -153,9 +155,10 @@ export const PropertyFilter: React.FC<IPropertyFilterProps> = ({
                   setFieldValue('name', getIn(newValues[0], 'value') ?? newValues[0]);
                 }}
                 ref={ref}
-                onBlur={(e: any) =>
-                  getIn(values, 'name') === '' && setFieldValue('name', e.target.value)
-                }
+                onBlur={(e: any) => {
+                  (getIn(values, 'name') === '' || getIn(values, 'name') !== e.target.value) &&
```

Comment:
do you need the first part of this conditional? the second check seems to encompass it.

Reply:
Good point, was getting lost in my logic lol, gonna test it out.